frontend/packages/dev-console/src/components/import/import-types.ts

```diff
@@ -302,32 +302,6 @@ export interface ImportData {
   buildStrategy: string;
   loader: LazyLoader<GitImportFormProps | SourceToImageFormProps>;
 }
-
-export enum TerminationTypes {
-  // t('devconsole~Edge')
-  edge = 'devconsole~Edge',
-  // t('devconsole~Passthrough')
-  passthrough = 'devconsole~Passthrough',
-  // t('devconsole~Re-encrypt')
-  reencrypt = 'devconsole~Re-encrypt',
-}
-
-export enum InsecureTrafficTypes {
-  // t('devconsole~None')
-  None = 'devconsole~None',
-  // t('devconsole~Allow')
-  Allow = 'devconsole~Allow',
-  // t('devconsole~Redirect')
-  Redirect = 'devconsole~Redirect',
-}
-
-export enum PassthroughInsecureTrafficTypes {
-  // t('devconsole~None')
-  None = 'devconsole~None',
-  // t('devconsole~Redirect')
-  Redirect = 'devconsole~Redirect',
-}
-
```

Comment:
I don't see a reason to remove our enums. We should keep then but revert that the values are i18n keys. But the types in this files should describe how our import "json" should look like.

Instead of removing this "unused" types we should use this types in the `interface TLSData`

And as they are not used before... Can you maybe change them to `TerminationType`, `InsecureTrafficType`, and `PassthroughInsecureTrafficType` (singular).